lessons/env-and-tools/README.md

```diff
@@ -1 +1,12 @@
-# put instructions here
+# How To Download Xcode
+
+1. From your Macbook, open the App Store.
+2. Search for *Xcode*.
+..* Note: Make sure you’re downloading Version 7.3.1. Do not download Xcode Beta 8 at this point. 
+3. Press ‘Get’ and then ‘Install App’. 
+4. You will now be prompted to provide your AppleID. If you do not have one, create one. Once you’ve provided a correct AppleID, click Buy.
+..*Note: Xcode is free. You do not have to make any purchases to download and use Xcode.
+4. When download is complete, open *Xcode* from your Applications folder. 
+5. The first time you open *Xcode* it’ll ask to install some additional components, allow it.
+6. Once complete, you should see a “Welcome to Xcode” screen. 
+7. Congratulations! You have successfully downloaded Xcode. Now you’re able to do the [pre-work assignment] (https://github.com/C4Q/AC3.2/blob/master/lessons/prework)
```

Comment:
This link got in there: (https://github.com/C4Q/AC3.2/blob/master/lessons/prework)

Is something doing this automatically?